test/plugin/scenarios/apm-toolkit-trace-scenario/src/main/java/test/org/apache/skywalking/apm/testcase/toolkit/controller/TestController.java

```diff
@@ -52,12 +52,12 @@ public String toolKitCase() {
         testService.testErrorMsg();
         testService.testErrorThrowable();
         testService.asyncCallable(() -> {
-            visit("http://localhost:8080/apm-toolkit-trace-scenario/case/asyncVisit/callable");
+            visit("http://localhost:8080/apm-toolkit-trace-scenario/case/asyncVisit?from=callable");
```

Comment:
Why change to the parameter?

Reply:
So the old one is not right, just because of the validator tool bug, it passed?

Reply:
So before operationName was unique

Reply:
Has been rolled back